### DISCUSSION TITLE
How to test my trained UNETR model

### DISCUSSION THREAD

**Discussion by ouyuxuanbridge** [Project-MONAI/MONAI#3866] [Q&A]
Hi contributors,

First I would like to thank you for this brilliant work.

I followed your instruction here(https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/unetr_btcv_segmentation_3d.ipynb), but I don't know how to test my model. Could you offer the code for test process in the tutorial?

Many thanks.

**Comment by ahatamiz** [comment]
Hi @Ouyuxuan623 

In the final block of the tutorial (_Check best model output with the input image and label_), we load the best model and perform test/inference and visualize the results.

Thanks

**Reply by btho733** [reply]
Hi @ahatamiz ,

I think the final block is still using the validation volumes. May I know if there is any code block performing inference on volumes from imageTs (as done in the spleen 3D segmentation tutorial https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb )?

Many Thanks

**Comment by ahatamiz** [comment]
Hi @btho733 

We used the validation volumes to compare against existing ground truth labels. But same can be applied to those volumes in imageTs which don't have labels.

**Reply by btho733** [reply]
Hi @ahatamiz ,

Thanks for the reply. I created an inference section based on the final sections of spleen 3D segmentation tutorial and tested on some of my data. It looks like it is working for me. 

Thanks
